backend/src/StamAcasa.IdentityServer/Startup.cs

```diff
@@ -74,13 +74,20 @@ public void ConfigureServices(IServiceCollection services)
                     );
                     break;
             }
+
+            services.AddSingleton(RabbitHutch.CreateBus(string.Format("host={0}:{1};username={2};password={3}",
```

Comment:
Do we need this anymore, as we are using the IQueueService implementation?

Reply:
Yes, the QueueService constructor needs a parameter of type IBus so the bus has to be created here to be resolved when QueueService is instantiated.